docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,1028 @@
+described in more detail below.
+
+All keys in the Z-JSON Instrument follow the lowerCamelCase naming convention. They also appear alphabetically
+inside each JSON object.
+
+## 2.1: Instrument Metadata
+
+```js
+{
+    "schema" : {
+        ...
+        "meta" : {
+            "createdBy": string,
+            "createdOn": string,
+            "createdWith": string,
+            "defaultLanguage": string,
+            "derivedFrom": string,
+            "importedFrom": string,
+            "includeMetaDataFields": boolean,
+            "instrumentVersion": string,
+            "lastUpdateOn": string,
+            "longName": {
+                ...
+            },
+            "multilingual": boolean,
+            "previousVersion": string,
+            "schemaVersion": string,
+            "shortName": string
+        },
+        ...
+    },
+    ...
+}
+```
+
+The metadata object contains useful information such as the version of the instrument, its name, its description, and
+some provenance data. The value of each key is defined as such:
+
+`meta.createdBy`: String.
+
+The developer or user responsible for creating the Z-JSON Instrument e.g. the Z-JSON Instrument of a paper instrument,
+created by some study, will be `createdBy` *this* LORIS developer.
+
+`meta.createdOn`: String.
+
+The date of creation of the Z-JSON Instrument. Each new version of an instrument will have a new `createdOn` date. Small
+changes, such as bug fixes, that do not require the creation of a new verion of an instrument should use `meta.lastUpdateOn`
+instead to track updates.
+                 
+`meta.createdWith`: String.
+
+The software or tool responsible for the creation of the ZJSON Instrument i.e. the Instrument Builder module or some conversion script.
+
+`meta.defaultLanguage`: Required String.
+                        
+The default (human communication) language of the instrument, represented by an IETF BCP 47 (RFC 5646 Standard) language tag e.g. "en-CA".
+It is important to set the default language as it is used as a key in other locations of the `schema` object. This key is required to be compatible with
+ReproNim's JSON-LD Instrument.
+
+`meta.derivedFrom`: String.
+                    
+The resource instrument that the Z-JSON Instrument derived from as a result of significant content modification such that the instrument is no longer the same                    
+instrument, and the creation of a new version does not suffice.
+                    
+This level of modification does not include format transcription i.e. linst to
+JSON, unless the content of the instrument has also been modified. The
+`meta.importedFrom` key should be used in the case of conversions.
+
+`meta.importedFrom`: String.
+
+The resource instrument that this Z-JSON Instrument was converted from, given
+that the imported content is preserved, and only its format is modified.
```

Comment:
if you convert a php or JSON instrument into Z-JSON. you refer to it here I think. if you later it, you keep this null and refer to the previous version in derived from

Reply:
How about something like:

```
The source that this instrument was converted from, if it was imported from another format and is otherwise unmodified. After any modifications to an imported instrument are made, this field should be set to the empty string.
```